### ISSUE TITLE
Render the Amplitudes chart

### ISSUE THREAD

**Issue by samreid** [phetsims/fourier-making-waves#8]
From https://github.com/phetsims/fourier-making-waves/issues/3, we would like to show an amplitude chart that looks roughly like this:

![image](https://user-images.githubusercontent.com/679486/98632344-92719980-22dc-11eb-8268-b02c49f3bf79.png)

The draggable bars have already been developed.  We need the chart background, tick labels, grid lines, etc.

In https://github.com/phetsims/fourier-making-waves/issues/3 I suggested that griddle would be able to accomplish this part.  I'll take a look.

**Comment by samreid** [comment]
Griddle has support for the chart area, tick labels, grid lines, and only showing vertical tick labels, etc, using XYChartNode.  I added support in the commit, and it looks like this:

![image](https://user-images.githubusercontent.com/679486/98637386-97871680-22e5-11eb-8873-7690391e4e04.png)

Some notes:
* This work revealed a bug in GridNode, mentioned and solved above. This bug would have bit us sometime, it was nice to find it earlier rather than later.
* I did not change the "Amplitudes" label which was already present
* The numeric labels don't exactly match the specification (-1, 0.5, 0, etc).  I think this looks better (easier to visually parse) but we can change the number formatter if desired.
* I use the chart bounds to determine the scale and spacing of the AmplitudeSliders.
* I think the numeric readouts should be done separately above the chart.  They can be aligned exactly with the AmplitudeSliders by using scenery coordinate transforms.
* @pixelzoom and I discussed over Zoom and he recommended to commit these changes.  He also indicated that the design for this part might change significantly.  Feel free to change/delete any of these parts as necessary.

**Comment by pixelzoom** [comment]
I see that you tacked on the x-axis label like this:

```js
        children: [ new HStrut( 75 ), xyChartNode, new Text( 'n', {
          fontSize: 14
        } ) ]
```

XYChartNode is responsible for axis tick marks.  Should it also be responsible for axis labels?

**Comment by samreid** [comment]
This seemed an unconventional way to label a chart (at least I couldn't think of any other occurrences like that).  But if we think this style will be widely used, let's add it to XYChartNode.

**Comment by pixelzoom** [comment]
What's unconventional about it?  

Axis labels should have alignment options, with default being to center on axis.

**Comment by samreid** [comment]
Thanks, I moved that recommendation to a new issue.

**Comment by pixelzoom** [comment]
@samreid said:
> The numeric labels don't exactly match the specification (-1, 0.5, 0, etc). I think this looks better (easier to visually parse) but we can change the number formatter if desired.

To clarify, the y-axis labels in the Java version (the previously the HTML version) look like this:

<img width="149" alt="screenshot_669" src="https://user-images.githubusercontent.com/3046552/98753500-b76c1800-2381-11eb-82f3-06d57a0d7e0d.png">

After @samreid's changes, the y-axis labels now look like this, where every labels has 1 decimal place:

<img width="122" alt="screenshot_670" src="https://user-images.githubusercontent.com/3046552/98753522-c783f780-2381-11eb-9dcd-380333cba715.png">

XYChartNode currently does not have a formatter, it has options for the number of decimal places in the labels. So this is not as simple as "change the number formatter" -- it requires changing the API for how labels are described.

In some previous sims, PhET has preferred to display integer values (especially 0) as integers.  But I'll leave that decision to the designers. @arouinfar is this change OK for Fourier?  And perhaps more importantly, is this an acceptable limitation for all charts?

**Comment by pixelzoom** [comment]
11/20/2020 design meeting with @arouinfar @kathy-phet @samreid @pixelzoom 

Over in https://github.com/phetsims/fourier-making-waves/issues/9#issuecomment-726391745, we decided to move in a different direction for Amplitudes, to make it less chart-like.   XYChartNode is feeling more and more like the wrong tool here, and may be replaced with something sim-specific.  

That said, if we continued to use XYChartNode, the consensus is:

* Integer values without decimals place (1, 0, -1) are preferred for this sim. @samreid pointed out that other sims (CCK?) show decimal places for these values. @kathy-phet and @arouinfar said that griddle needs to be flexible about format.  Tracking in https://github.com/phetsims/griddle/issues/84#issuecomment-726395222.

* Charts (in general) need to support axes and tick marks.  In this case, the horizontal '0' gridline should be darker, and there should be tick marks that are darker than the grid lines. Tracking in https://github.com/phetsims/griddle/issues/84#issuecomment-726395222.
